src/data/inebriety.txt

```diff
@@ -5,8 +5,8 @@
 # Drink	Inebriety	Level Req	quality	Adv	Musc	Myst	Moxie
 
 [experimental crimbo booze]	3	1	good	0	0	0	0	Random
-1950 Vampire Vintner wine	1	1	awesome	5	0	0	0	Unspaded, WINE
 5-hour acrimony	5	1	awesome	0	0	0	0	+10 PvP fights
+1950 Vampire Vintner wine	1	1	awesome	5	0	0	0	Unspaded, WINE
```

Comment:
same here, I think it's sorting starting numbers as numbers instead of alphabetically?

Reply:
Yes. I did that on purpose - elsewhere in the data we expect it to be so.